gym-unity/gym_unity/envs/__init__.py

```diff
@@ -102,9 +102,9 @@ def __init__(
         self._previous_decision_step = decision_steps
 
         # Set action spaces
-        if self.group_spec.is_action_discrete():
-            branches = self.group_spec.discrete_action_branches
-            if self.group_spec.action_size == 1:
+        if self.group_spec.action_spec.is_discrete():
```

Comment:
General question - will gym support hybrid action spaces, or do we need to keep it all-continuous/all-discrete?

Reply:
Gym will support it (you can have your action space be a list of other action spaces). But I do not think there are a lot of environments that do that.